kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -139,6 +142,24 @@
                         ++orderInGroup,
                         Width.LONG,
                         "S3 accelerated endpoint enabled");
+
+      CONFIG_DEF.define(S3_PROXY_HOST,
+              Type.STRING,
+              Importance.HIGH,
+              "proxy host.",
+              group,
+              ++orderInGroup,
+              Width.LONG,
+              "proxy host");
+
+      CONFIG_DEF.define(S3_PROXY_PORT,
+              Type.INT,
+              Importance.HIGH,
+              "proxy port.",
+              group,
+              ++orderInGroup,
+              Width.LONG,
```

Comment:
Width can probably be SHORT here -- ports should only be a few digits long at most.